### ISSUE TITLE
document shibboleth / ADFS configuration

### ISSUE THREAD

**Issue by pameyer** [IQSS/dataverse#5490]

**Comment by pdurbin** [comment]
@pameyer in 14f4f344f I added the tip we talked about... where the ADFS server hostname should go. I still think it might be nice to include a sanitized version of the working `MetadataProvider` file you're using, if possible. Awesome that you got this working!

I guess one more thought is that perhaps we could mention ADFS or Active Directory somewhere in a future version of http://guides.dataverse.org/en/4.10.1/installation/config.html#auth-modes-local-vs-remote-vs-both . That "auth modes" section could probably stand to be revisited anyway. Maybe we could just change...

"To configure Shibboleth see the Shibboleth section"

... to ...

"To configure Shibboleth or Active Directory/ADFS see the Shibboleth section"

?

I don't know. I'm not sure what level of support we intend to give this. Auth is important. As we discussed, we know that Julio wants to use Active Directory for auth: http://irclog.iq.harvard.edu/dataverse/2019-01-25#i_85555 . I have his email address. Maybe we should ask him to try it? He said "I must accomplish this setup in order to implement dataverse here."
